spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -82,18 +81,19 @@ trait CommonJobExecution {
   protected def prepareJob[T]()
                              (implicit dao: MenasDAO,
                               cmd: JobConfigParser[T],
-                              fsUtils: DistributedFsUtils,
                               spark: SparkSession): PreparationResult = {
     val confReader: ConfigReader = new ConfigReader(conf)
     confReader.logEffectiveConfigProps(Constants.ConfigKeysToRedact)
-
     dao.authenticate()
+
+    implicit val hadoopConf = spark.sparkContext.hadoopConfiguration
```

Comment:
I'm thinking if this would make more sense to be a field

Reply:
field of what? Do you mean a non-implicit parameter of the method `prepareJob[T]()`?